thrift/rpc.thrift

```diff
@@ -122,10 +122,12 @@ service rpc_service {
   /** Query ops **/
   // Write ops
   i64 append(1: i64 multilog_id, 2: binary data),
+  i64 append_json(1: i64 multilog_id, 2: binary data),
```

Comment:
`binary` -> `string`

Reply:
note: for the read_json return value as well